### ISSUE TITLE
[Feature] A model and some post process.

### ISSUE THREAD

**Issue by 610265158** [open-mmlab/mmpose#2345]
### What is the feature?

Hi， I am the author of https://github.com/610265158/Peppa_Pig_Face_Landmark, and there is a simple model with SOTA result on WFLW（ nme 3.90), and also some post process, would you like to merge the pr, if i submit it?

**Comment by Ben-Louis** [comment]
Absolutely! We greatly appreciate your anticipated contribution. Should you have any questions or concerns, don't hesitate to contact us. This [contribution guide](https://mmpose.readthedocs.io/en/latest/contribution_guide.html) might also be helpful.

**Comment by Tau-J** [comment]
In addition, we provide 2 appoaches to contribute. First, directly contribute to our core package, which requires unit test. Second, you can also submit to `projects`, which requires no unit test. FYI.

**Comment by 610265158** [comment]
That would be great, i will submit it when it finished.

**Comment by 610265158** [comment]
Hi,  i reproduce the result 3.94 nme with wflw, a little bit lower than my model, but still sota. 
I have sub the pr. 
And i have some question.
1. How to get the nme score with wflw testset, and its sub test set.
2. If the pr is merged how could i sub the pretrained model and the log file.
3. And i use regresssion wingloss with a nchw shape tensor,  but it reduce as torch.mean(torch.sum(losses, dim=[1, 2]), dim=0), do we need a more flexible way to do the reduce mean or sum.

**Comment by 610265158** [comment]
Hi,  i reproduce the result 3.94 nme with wflw, a little bit lower than my model, but still sota. 
I have sub the pr. 
And i have some question.
1. How to get the nme score with wflw testset, and its sub test set.
2. If the pr is merged how could i sub the pretrained model and the log file.
3. And i use regresssion wingloss with a nchw shape tensor,  but it reduce as torch.mean(torch.sum(losses, dim=[1, 2]), dim=0), do we need a more flexible way to do the reduce mean or sum.
